test/spec/actions/delete_way.js

```diff
@@ -8,7 +8,7 @@ describe('iD.actionDeleteWay', function() {
 
     it('removes a way from parent relations', function() {
         var way      = iD.osmWay(),
-            relation = iD.osmRelation({members: [{ id: way.id }, { id: 'w-2' }]}),
+            relation = iD.osmRelation({members: [{ id: way.id }, { id: 'w-99' }]}),
```

Comment:
this test sporadically fails if `way.id` is also allocated ID #&shy;2. Changing it to `w-99` prevents this edge case.